indico/modules/events/abstracts/forms.py

```diff
@@ -125,6 +125,9 @@ class AbstractSubmissionSettingsForm(IndicoForm):
                                                     enum=SubmissionRightsType, sorted=True,
                                                     description=_('Specify who will get contribution submission rights '
                                                                   'once an abstract has been accepted'))
+    allow_submitters_edit_contributions = BooleanField(_('Allow submitters edit contributions'), widget=SwitchWidget(),
```

Comment:
Should this setting really be in the CfA? It's not really related to abstracts...